src/components/input/bl-input.ts

```diff
@@ -17,6 +17,9 @@ export type InputSize = 'small' | 'medium' | 'large';
 /**
  * @tag bl-input
  * @summary Baklava Input component
+ *
+ * @cssproperty [--bl-input-padding-left] Sets the padding left
+ * @cssproperty [--bl-input-padding-right] Sets the padding right
```

Comment:
We'll need to start considering right-to-left languages soon, so let's start using logical property naming here like `--bl-input-padding-start` and `--bl-input-padding-end`. You can update the description accordingly.